lib/internal/modules/cjs/loader.js

```diff
@@ -554,6 +555,8 @@ Module._load = function(request, parent, isMain) {
   try {
     module.load(filename);
     threw = false;
+  }catch(err){
+    throw new ESM_CJS_LOAD_ERROR(err);
   } finally {
```

Comment:
is ther a reason to have the `finally` here, since we catch it? this way we can remove thee `threw` variable.

Reply:
Yes the `finally` will still run, and is important to clear the cache for errors.